R/lassnags.r

```diff
@@ -0,0 +1,246 @@
+# ===============================================================================
+#
+# PROGRAMMERS:
+#
+# andrew.sanchezmeador@nau.edu - https://github.com/bi0m3trics
+# jean-romain.roussel.1@ulaval.ca  -  https://github.com/Jean-Romain/lidR
+#
+# COPYRIGHT:
+#
+# Copyright 2016 Jean-Romain Roussel
+#
+# This file is part of lidR R package.
+#
+# lidR is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+# ===============================================================================
+
+#' Snag classification of airborne LiDAR point clouds using neighborhood metrics
+#'
+#' Snag clasification using the algorithim proposed by Wing et al 2015 (see details). The function
+#' attributes to each point of the point cloud a number identifying a snag class (\code{snagCls} column).
```

Comment:
`snagID` would be more constitant with `treeID`, `pulseID`, `flighlineID`. What do you think?

Reply:
At present, it's just a classification algorithm (wing et al), so I left it as snagCls but included in the doc the possibility of attributing snagID in future algorithms